lib/views/configuration-view.js

```diff
@@ -79,6 +85,10 @@ export default class ConfigurationView extends View {
       self.close();
     });
 
+    importButton.addEventListener('click', (event) => {
+      Import.import(self);
```

Comment:
Display notification in case of error or sucess.

Reply:
Done it in import class.